seqr/models.py

```diff
@@ -608,19 +608,18 @@ class VariantTag(ModelWithGUID):
     search_parameters = models.TextField(null=True, blank=True)  # aka. search url
 
     def __unicode__(self):
-        return "%s:%s" % (str(self.saved_variant), self.variant_tag_type.name)
+        return "%s:%s" % (str(self.saved_variants), self.variant_tag_type.name)
```

Comment:
does this work or does this create something with weird brackets in the guid or something?